src/Sulu/Bundle/ContentBundle/Search/Metadata/StructureProvider.php

```diff
@@ -244,7 +244,7 @@ public function getMetadata(Metadata $documentMetadata, StructureMetadata $struc
             [
                 'type' => 'string',
                 'stored' => true,
-                'indexed' => false,
+                'indexed' => true,
```

Comment:
The problem with this solution is that `structureType` is now included in every search, i.e. if you search for `default` you won't only find the pages containing `default` in its content, but also the pages with the `default` template (btw, we currently have the same issue with the status, searching for `published` will return every page in the system).

We would have to adapt the queries, but there is not really a consistent way. The problem is that all the content fields are aggregated in the `ZendLuceneAdapter` to a field called `__content`, so we could adapt the query in the [SearchController](https://github.com/sulu-io/sulu-standard/blob/develop/src/Client/Bundle/WebsiteBundle/Controller/SearchController.php#L53) to only search for that field, but the problem is that this aggregation is not done in the `ElasticSearchAdapter`, so the query would not work anymore for this search engine.

@chirimoya How should we handle this? Should we live with the "expanded" search results for now, and investigate that problem later?